resource_catalogue_fastapi/__init__.py

```diff
@@ -196,3 +228,41 @@ async def update_item(
     producer.send((json.dumps(output_data)).encode("utf-8"))
 
     return JSONResponse(content={"message": "Item updated successfully"}, status_code=200)
+
+
+@app.post(
+    "/catalogs/user-datasets/{workspace}/ordered-data", dependencies=[Depends(opa_dependency)]
+)
+async def order_item(
+    request: Request,
+    workspace: str,
+    item_request: ItemRequest,
+    producer=Depends(get_producer),  # noqa: B008
+):
+    """Endpoint to create a new item and collection within a workspace with an order status, and
+    execute a workflow to order the item"""
+
+    authorization = request.headers.get("Authorization")
```

Comment:
```suggestion
    authorisation = request.headers.get("Authorization")
```

Feel free to ignore this suggestion if you don't agree!

Reply:
I would generally prefer to stick to English spelling but for this one it is specifically "Authorization" in the header, so I think better to be consistent with that